### DISCUSSION TITLE
How to bind a command to the keyboard key "/" ?

### DISCUSSION THREAD

**Discussion by gaberiel44** [d2phap/ImageGlass#1612] [Q&A] [answered]
I tried:

    "MnuGoToFirst": [
      "/"
    ],

but it gets turned into:

    "MnuZoomIn": [
      "?"
    ],

Then when I run IG again it gets turned into:
````
         "MnuZoomIn": [
            "None"
            ],
````

I tried looking for the correct keyword in the code base here, I did not find it. Thanks!

**Comment by d2phap** [comment, marked as answer]
The hotkey uses [Winforms Keys enum](https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.keys?view=windowsdesktop-7.0) with some basic key mapping:
https://github.com/d2phap/ImageGlass/blob/c65f5989dc89de39bc5cfdcc9d83d41aba5492ae/v9/Components/ImageGlass.Base/Types/HotKey.cs#L55-L79

You may use `OemQuestion` for `/` with US keyboard layout.

The upcoming v9.0 beta 5 will have a better keyboard parser, and a hotkey picker window.

![image](https://github.com/d2phap/ImageGlass/assets/3154213/797292e9-2cb9-4a9d-a9f1-924a14621ecf)

**Reply by gaberiel44** [reply]
This is great news! looking forward to the next update! cheers
